ui/tests/unit/adapters/pki/tidy-test.js

```diff
@@ -38,24 +38,43 @@ module('Unit | Adapter | pki/tidy', function (hooks) {
       safetyBuffer: '120h',
       backend: this.backend,
     };
-    await this.store
-      .createRecord('pki/tidy', this.payload)
-      .save({ adapterOptions: { tidyType: 'manual-tidy' } });
+    await this.store.createRecord('pki/tidy', this.payload).save({ adapterOptions: { tidyType: 'manual' } });
```

Comment:
Not sure if it's worth calling out here, but the `save()` method goes to the right endpoint depending on whether or not the `pki/tidy` record `isNew` or not. 

I want to avoid any confusion that passing `tidyType` into the `adapterOptions` influences which endpoint is used, `tidyType` is more of a safeguard because it just surfaces a warning to the developer that they're using the wrong record type